client/src/Overview/CartManagement.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+const CartManagement = ({ }) => {
+
+  return (
+    <div id="cartManagement">
+      <h3>Cart Management</h3>
+      <select>
+        <option>Select Shirt Size</option>
```

Comment:
These options may need to be dynamic. I don't think all the products have the same styles/sizes